pkg/vulnsrc/nvd/nvd.go

```diff
@@ -120,13 +121,48 @@ func (vs VulnSrc) commit(tx *bolt.Tx, items []Item) error {
 	return nil
 }
 
-func (vs VulnSrc) save(items []Item) error {
+func (vs VulnSrc) save(cves []Cve) error {
 	log.Println("NVD batch update")
 	err := vs.dbc.BatchUpdate(func(tx *bolt.Tx) error {
-		return vs.commit(tx, items)
+		return vs.commit(tx, cves)
 	})
 	if err != nil {
 		return xerrors.Errorf("error in batch update: %w", err)
 	}
 	return nil
 }
+
+// getCvssV2 selects vector, score and severity from V2 metrics
+func getCvssV2(metricsV2 []CvssMetricV2) (score float64, vector string, severity types.Severity) {
+	for _, metricV2 := range metricsV2 {
+		// save first metric or the `Primary` metric if `Secondary` metric was saved previously
+		if score == 0 || metricV2.Type == primaryType {
```

Comment:
If I understand correctly, the type doesn't matter. We need metrics from NVD, so I think we should take metrics with `source: nvd@nist.gov`.

>source identifies the organization that provided the metrics information and type identifies whether the organization is a primary or secondary source. Primary sources include the NVD and CNA who have reached the provider level in CVMAP. 10% of provider level submissions are audited by the NVD. If a submission has been audited the NVD will appear as the primary source and the provider level CNA will appear as the secondary source.

```
"cvssMetricV30": [
                        {
                            "source": "secalert@redhat.com",
                            "type": "Secondary",
                            "cvssData": {
                                "version": "3.0",
                                "vectorString": "CVSS:3.0/AV:P/AC:H/PR:N/UI:N/S:U/C:N/I:L/A:H",
                                "attackVector": "PHYSICAL",
                                "attackComplexity": "HIGH",
                                "privilegesRequired": "NONE",
                                "userInteraction": "NONE",
                                "scope": "UNCHANGED",
                                "confidentialityImpact": "NONE",
                                "integrityImpact": "LOW",
                                "availabilityImpact": "HIGH",
                                "baseScore": 4.8,
                                "baseSeverity": "MEDIUM"
                            },
                            "exploitabilityScore": 0.5,
                            "impactScore": 4.2
                        }
                    ],
```

Reply:
Thanks! fixed in https://github.com/aquasecurity/trivy-db/pull/374/commits/96b8f7d8cbc3446b5807f6815edaa0c705dbdb52